docs/deployment.md

```diff
@@ -0,0 +1,19 @@
+## How to deploy
+
+This project uses the following [Jenkins file](https://github.com/empathyco/platform-jenkins-library/blob/master/vars/XComponentsPipeline.groovy).
+To deploy the project in the different environments you should take this into account:
+
+| Environment | Branch     | Tag       |
+|-------------|------------|-----------|
+| test        | any branch | test      |
+| staging     | main       |           |
+| production  | main       | release-* |
+
+So everytime that a commit is made in main (merge PR, commit directly on main...etc), the changes are going to be
```

Comment:
What do you think about something like?:

Every time a commit is made to main, the changes are going to be deployed to `staging`.
To deploy to `test` or `production` a tag must be created.

Reply:
A la segunda parte le añadiría un "correspondent tag" por claridad.

To deploy to `test` or `production` the correspondent tag must be created.